### ISSUE TITLE
Replay a series of strokes

### ISSUE THREAD

**Issue by kotpal** [literallycanvas/literallycanvas#396]
A cursory glance of the documentation doesn't show that LiterallyCanvas supports replaying of strokes. This is useful to show a series of strokes in a drawing - instead of loading all the strokes and displaying at one.

**Comment by irskep** [comment]
You're correct. But as a general note, I am very unlikely to ever put time into this feature. Also, you could probably do it without actually changing the library, i.e. just on the user end.
